README.md

```diff
@@ -87,13 +87,21 @@ Client side highlighting does not require pygments to be installed. This will us
 
 ### Disqus support
 
-To use this feature, uncomment and fill out the `disqusShortname` parameter in `config.toml`.
+To use this feature add your disqus shortname to the hugo.toml file like this:
+
+```toml
+[services]
+  [services.disqus]
+    shortname = ''
```

Comment:
```suggestion
[services.disqus]
shortname = ''
```

Probably matches everything else, so not changing it here, but I really don't like this fake TOML nesting. Pretending TOML cares about indentation when it doesn't isn't helpful, IMO.